db/migrate/20250710180401_copy_ai_summarization_model_to_persona_default.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+class CopyAiSummarizationModelToPersonaDefault < ActiveRecord::Migration[7.2]
+  def up
+    ai_summarization_model =
+      DB.query_single("SELECT value FROM site_settings WHERE name = 'ai_summarization_model'").first
+
+    if ai_summarization_model.present? && ai_summarization_model.start_with?("custom:")
+      # Extract the model ID from the setting value (e.g., "custom:-5" -> "-5")
+      model_id = ai_summarization_model.split(":").last
+
+      # Update the summarization personas (IDs -11 and -12) with the extracted model ID
+      execute(<<~SQL)
+        UPDATE ai_personas
+        SET default_llm_id = #{model_id}
+        WHERE id IN (-11, -12) AND default_llm_id IS NULL
```

Comment:
We need to make sure this also works for sites using custom personas by adding the IDs from:

- ai_summarization_persona
- ai_summary_gists_persona